### PR TITLE
tests: Add event-enabled tests over SRD

### PR DESCRIPTION
**Pull request by ygoldamzn**
Added event enabled cq tests over SRD, with new and old send.

### CODE REVIEW

**Review by EdwardSro — CHANGES_REQUESTED** [review]
Please move the test to test_efa_srd.py or another EFA-specific test file

**Comment by ygoldamzn** [comment]
> Please move the test to test_efa_srd.py or another EFA-specific test file

Done.
I've decided to duplicate the create_cq_with_comp_channel() method because:
1. sharing between test suites introduces a bad dependency since there's no real sense of hierarchy there
2. it seems too specific to move to a base library

File: tests/test_efa_srd.py [hunk 1]
```diff
@@ -12,6 +13,29 @@
...
+        self.cq = CQ(self.ctx, self.num_msgs, None, self.comp_channel)
+        self.cq.req_notify()
+
+
+class CqEventsSRDTestCase(EfaRDMATestCase):
+    def setUp(self):
+        super().setUp()
+        self.iters = 100
+        self.server = None
+        self.client = None
```
Comment (EdwardSro) [inline tests/test_efa_srd.py:31]: BTW, no need to initiate server and client as it's already done in RDMATestCase.__init__()
Except for that, it looks good to me.

**Review by EdwardSro — APPROVED** [review]

File: tests/test_efa_srd.py (hunk 1 above)
Comment (ygoldamzn) [reply tests/test_efa_srd.py]: OK, removed.

**Review by EdwardSro — APPROVED** [review]

**Pull request merged**